application.py

```diff
@@ -19,8 +20,12 @@ def index():
             # TODO: show there's an error on the page
             return render_template('base.html')
         else:
-            r = requests.post('{}?language={}'.format(app.config['IMAGE_PROCESSOR'], request.args.get('language')),
-                              files={'file': file.read()})
+            url = '{}?language={}'.format(app.config['IMAGE_PROCESSOR'], request.args.get('language'))
```

Comment:
Might be nice to create a `url_builder`. Given the base url and request.args it will add everything from the dictionary into the request.

Reply:
Given that I want to move to React over Christmas, I don't see much point in creating a util which we will end up not using.

Reply:
True